core/src/main/scala/kafka/utils/PasswordEncoder.scala

```diff
@@ -20,12 +20,12 @@ import java.nio.charset.StandardCharsets
 import java.security.{AlgorithmParameters, NoSuchAlgorithmException, SecureRandom}
 import java.security.spec.AlgorithmParameterSpec
 import java.util.Base64
-
```

Comment:
small nit: did we mean to remove this new line?

Reply:
Almost certainly done by my IDE, I'll fix it